ApeSwapFinance/apeswap-frontend
# Analytics Enhancements - USD Value

Related to [Analytics Enhancements - USD Value](https://app.clickup.com/t/383uyxg) task
This PR includes:
- USD Value in `farm:stake` event
- USD Value in `vault:stake` event
- USD Value in `jungle_farm:stake` event
- USD Value in `gnana:buy` and `gnana:sell` event
- USD Value in `iao:buy` and `iao:claim` event
- USD Value in `zap:liquidity` event
- USD Value in `migrate_liq` event
- Created `zap:farm` event which includes USD Value
- Created `orders` event which includes USD Value

How to test:
1. Run locally or use the preview link
2. Do the action to test (i.e. Stake in a Farm)
3. Check if the object in the `dataLayer` has the corresponding `usdAmount`
```
dataLayer.filter(d => d.event == "farm" && d.cat == "stake")
```
